ca/certificate-authority.go

```diff
@@ -17,21 +19,41 @@ import (
 	"github.com/letsencrypt/boulder/policy"
 
 	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/auth"
-	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/config"
+	cfsslConfig "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/config"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/helpers"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/ocsp"
 	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/signer"
 	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/signer/remote"
 )
 
+type Config struct {
+	Server       string
+	AuthKey      string
+	Profile      string
+	TestMode     bool
+	DBDriver     string
+	DBName       string
+	SerialPrefix int
+	// A PEM-encoded copy of the issuer certificate.
+	IssuerCert string
+	// This field is only allowed if TestMode is true, indicating that we are
+	// signing with a local key. In production we will use an HSM and this
+	// IssuerKey must be empty (and TestMode must be false). PEM-encoded private
+	// key used for signing certificates and OCSP responses.
+	IssuerKey string
```

Comment:
We decided that since OCSP signing happens on the CA box and is offline, there wasn't significant benefit to signing OCSP responses with a different key. And using a different key for OCSP responses does have a significant detriment: You need to include an extra cert along with OCSP response (both stapled and non-stapled). This can add a lot of bytes to a handshake.

In future changes, I'll add support for signing OCSP responses with a PKCS#11 key.